### ISSUE TITLE
Client connect trigger instead of min threshold

### ISSUE THREAD

**Issue by Vukodlak7** [vmorganp/Lazytainer#10]
I got everything working great, but is there any way to make it spin up the docker container based on when a client connects or sends an http request to the port?

I am using it with motioneye, and motioneye doesn't ever really settle down to 0 packets and is impossible to gauge the threshold correctly.  It does, however, seem to register the clients connected just fine.   Also, you cannot send enough traffic to motioneye to get it to trigger back on because of the login screen it uses, I think.

TIA!

**Comment by Arson31** [comment]
I agree some containers are always communicating with one another and it's almost impossible to get a good baseline of how many packets are too little or too much to start a container.

**Comment by calrock27** [comment]
This is an incredible idea for game servers among friends! I'm looking at using this for a Satisfactory server (very high memory utilization) but am running into the same issue described here. The container is nearly immediately restarting based on received packets despite no players being logged in. I suspect this is some sort of keep alive traffic potentially coming from Steam and/or Epic? I'll look to mitigate this by watching averages of rx packets in the logging. 

Is a clients connected metric for something like this even feasible for restarting a container? Clients connected will always be zero when stopped, and they aren't "connected" until the container is running and they're logged in. 

Excellent work here either way!

**Comment by vmorganp** [comment]
I've been hacking on this project a bunch over the last couple of days. 

Something I've come across (and would love to be proven wrong on) is that in order to make a client connect trigger work, all traffic would need to be something where we could determine what a "client" is.

Because this was purpose built to be a lazy loading front for containers as a whole and not for something like web servers specifically, or game servers specifically, I don't know that it will be possible to determine "client connection".

Again, if you know a way to determine that for generic traffic, please let me know, but I just don't know if this issue is solvable without compromising the purpose of lazytainer or a TON of custom code. 

That said, I'm actively developing port specific packet thresholds which should help with some of the "spammy traffic hit lazytainer and now your other container is awake" stuff that was happening.
